releases/v2.0-beta.20180326.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What&#39;s New in v2.0-beta.20180326
+toc: false
+summary: Additions and changes in CockroachDB version v2.0-beta.20180326 since version v2.0-beta.20180319
+---
+
+## March 26, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180326.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180326.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180326.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180326.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### General Changes
+
+- A CockroachDB process now flushes its logs upon receiving `SIGHUP` instead of `SIGUSR1` as it did earlier. This is aimed to simplify the automation of process monitoring, test, and backup tools. [#23783][#23783]
+- Information about [zone config](../v2.0/configure-replication-zones.html) usage is now included in [diagnostic reports](../v2.0/diagnostics-reporting.html). [#23750][#23750]
+
+### Enterprise Edition Changes
+
+- Added the `cloudstorage.timeout` [cluster setting](../v2.0/cluster-settings.html) for import/export operations. [#23776][#23776]
+
+### SQL Language Changes
+
+- SQL features introduced in CockroachDB v2.0 cannot be used in clusters that are not [upgraded fully to v2.0](../v2.0/upgrade-cockroach-version.html). [#24013][#24013]
+- Added an `escape` option to the `encode()` and `decode()` [built-in functions](../v2.0/functions-and-operators.html). [#23781][#23781]
+- Introduced a series of PostgreSQL-compatible, privilege-related [built-in functions](../v2.0/functions-and-operators.html). [#23839][#23839]
+- Added the `pg_language` table to the `pg_catalog` virtual schema. [#23839][#23839]
+- Added the `anyarray` type to the `pg_type` catalog. [#23836][#23836]
```

Comment:
pg_type is a virtual schema, not a catalog.

Reply:
Thanks, @knz. Updated in https://github.com/cockroachdb/docs/pull/2813.